consul/src/main/java/com/linecorp/armeria/internal/consul/ConsulClient.java

```diff
@@ -75,53 +75,72 @@ ObjectMapper getObjectMapper() {
     }
 
     /**
-     * Registers a service to Consul Agent with service ID.
+     * Registers a service to Consul Agent.
      *
      * @param serviceId a service ID that identifying a service
      * @param serviceName a service name to register
      * @param endpoint an endpoint of service to register
      * @param check a check for the service
      * @param tags tags for the service
-     * @return a {@link CompletableFuture} that will be completed with the registered service ID
+     * @return an HttpResponse representing the HTTP response from Consul
```

Comment:
Let's put an empty line:
```suggestion

     * @return the {@link HttpResponse} from Consul
```